app/views/content/_form_for_taxons.html.erb

```diff
@@ -0,0 +1,7 @@
+<h3>Taxonomy</h3>
+
+<%= f.select :taxons,
+    linkables.taxons,
+    {},
+    { multiple: true, class: "select2", data: { placeholder: "Choose taxons..." } } %>
```

Comment:
Could we make the explain text:

> This taxonomy is currently being developed by the Finding Things team.

Also, we should probably change "Choose taxons" to "Choose one..." or something like that. We shouldn't use "taxon" in the UI (all tough we don't have a better name yet - https://trello.com/c/KoydaDC8).